### PR TITLE
Added support for http.HTTPStatus in HTTPResponse

### PR DESCRIPTION
**Pull request by metamoof**
It should be possible to pass values from the standard library http.HTTPStatus enum to HTTPResponse

### CODE REVIEW

**Comment by YunchuWang** [comment]
LGTM, can you fix the test failures and add unit test for it?

**Comment by metamoof** [comment]
@microsoft-github-policy-service agree

File: azure/functions/_http.py [hunk 1]
```diff
@@ -52,6 +53,7 @@ class HttpResponse(_abc.HttpResponse):
 
     :param int status_code:
```
Comment (metamoof) [inline azure/functions/_http.py:54]: I'm not sure hot to indicate `int` or `http.HTTPStatus` here

**Comment by vrdmr** [comment]
Thanks a lot, @metamoof for the PR. We'll test and get this in asap. This is definitely a useful addition.

**Review by vrdmr — APPROVED** [review]

**Review by gavin-aguiar — APPROVED** [review]

**Pull request merged**